Update ApiURL information (#314)
DataDog/datadog-cloudformation-resources: README.md

    }
  }
  ```

>    Specify the `ApiURL` if you are using a non-US1 account (defaults to `https://api.datadoghq.com`). For example, use `https://api.datadoghq.eu` for an EU account.

Reviewer: ```suggestion
   Specify the `ApiURL` if you are using an account in a region other than US1. For example, use `https://api.datadoghq.eu` for an EU account.
```

> 7. After you have your resource configured, [create your AWS stack][3] that includes any of the activated Datadog resources.

Reviewer: ```suggestion
7. After you have configured your resource, [create your AWS stack][3] that includes any of the activated Datadog resources.
```
